core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -6764,4 +6769,25 @@ protected MyProjectRule(Config config) {
         .checkUnchanged();
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-4192">[CALCITE-4192]
+   * fix aggregate column origins searching by RelMdColumnOrigins</a>. */
+  @Test void testColumnOriginAfterAggProjectMergeRule() {
+    final String sql = "select count(ename), SAL from emp group by SAL";
```

Comment:
The primary target of this test is related to RelMetadata, so it should be moved to RelMetadataTest?

Reply:
thanks, good idea!